webapp-django/crashstats/signature/views.py

```diff
@@ -374,7 +375,12 @@ def signature_correlations(request, params):
         elif all("esr" in version for version in params["version"]):
             context["channel"] = "esr"
 
-    context["product_name"] = "Firefox"
+    default_product_name = productlib.get_default_product().name
+    product_name = params.get("product", default_product_name)
+    if isinstance(product_name, (list, tuple)):
+        product_name = product_name[0]
```

Comment:
This un-hard-codes the product so if we ever add correlations support for products other than Firefox, it'll work correctly without us having to fix a bunch of hard-coded places.